### PR TITLE
Added shortcuts for mostly used actions

### PR DESCRIPTION
**Pull request by vi4m**
`make run` will run server
`make menu` will load menudata

### CODE REVIEW

**Comment by szok** [comment]
:ok:

File: Makefile [hunk 1]
```diff
@@ -29,3 +29,9 @@ docs:
 	cd ./docs && make html
 
 coveralls: docs coverage
+
+run:
+	dev_ralph runserver_plus 0.0.0.0:8000
+
+menu:
+	dev_ralph sitetree_resync_apps
```
Comment (ar4s) [inline Makefile]: `ralph` instead of `dev_ralph` cause `dev_ralph` required dev's packages

**Comment by mkurek** [comment]
@vi4m could you please make it mergeable and fix @ar4s notice?

**Comment by ar4s** [comment]
:ok:

**Comment by szok** [comment]
:ok:

**Comment by vi4m** [comment]
Yes please.

**Pull request merged**